producer.py

```diff
@@ -1,11 +1,34 @@
-from json_asset import scrape
+from scraper import scrape, current_time
+from area_dict import area
 
-def producer():
-    raw_json = scrape('C03')
+current_time = current_time()
 
-    events = {}
+def producer(): #iterating each area in area_dict and returns a dictionary of all areas
+    all_areas = {}
+
+    for area_name in area.values():
+        area_code = area[area_name]
+        all_areas[area_code] = json_rearranger(area_code)
```

Comment:
Your  "area" was defined as 
{"道路名" : "コート"}, but you iterate thought "area.values" and named it "area_name".  Thus I'm sure the "area_code" will be NA since you're searching the dict with the value, not the key. Have you tested this function yet?